File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/03-additional/04.transportPrice.js
function transportPrice(input) {

    const startPriceTaxi = 0.70;
    const dailyRateTaxiPerKm = 0.79;
    const nightRateTaxiPerKm = 0.90;
    const busRatePerKm = 0.09;
    const trainRatePerKm = 0.06;

    const nKm = Number(input[0]);
    const dayOrNight = input[1];

    let price = 0;

    if (nKm >= 100) {
        price = trainRatePerKm * nKm;
    } else if (nKm >= 20) {
        price = busRatePerKm * nKm;
    } else if (nKm < 20 && dayOrNight === "day") {
        price = startPriceTaxi + dailyRateTaxiPerKm * nKm;
    } else {
        price = startPriceTaxi + nightRateTaxiPerKm * nKm;
    }

    console.log(price.toFixed(2));
}
transportPrice(["25", "day"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/03.additional/04.transport_price.py
startPriceTaxi = 0.70
dailyRateTaxiPerKm = 0.79
nightRateTaxiPerKm = 0.90
busRatePerKm = 0.09
trainRatePerKm = 0.06

nKm = int(input())
dayOrNight = input()

price = 0

if nKm >= 100:
    price = trainRatePerKm * nKm
elif nKm >= 20:
    price = busRatePerKm * nKm
elif nKm < 20 and dayOrNight == "day":
    price = startPriceTaxi + dailyRateTaxiPerKm * nKm
else:
    price = startPriceTaxi + nightRateTaxiPerKm * nKm

print(f'{price:.2f}')
